File: app/src/main/java/me/tsukanov/counter/SharedPrefKeys.java
package me.tsukanov.counter;

import androidx.annotation.NonNull;

public enum SharedPrefKeys {
  ACTIVE_COUNTER("activeKey"),
  KEEP_SCREEN_ON("keepScreenOn"),
  LABEL_CONTROL_ON("labelControlOn"),
  HARDWARE_BTN_CONTROL_ON("hardControlOn"),
  VIBRATION_ON("vibrationOn"),
  SOUNDS_ON("soundsOn"),
  THEME("theme");

  @NonNull private final String name;

  SharedPrefKeys(@NonNull final String name) {
    this.name = name;
  }

  @NonNull
  public String getName() {
    return name;
  }
}

File: app/src/main/java/me/tsukanov/counter/SharedPrefKeys.kt
package me.tsukanov.counter

enum class SharedPrefKeys(val key: String) {
    ACTIVE_COUNTER("activeKey"),
    KEEP_SCREEN_ON("keepScreenOn"),
    HIDE_CONTROLS("hideControls"),
    HIDE_LAST_UPDATE("hideLastUpdate"),
    LABEL_CONTROL_ON("labelControlOn"),
    HARDWARE_BTN_CONTROL_ON("hardControlOn"),
    VIBRATION_ON("vibrationOn"),
    SOUNDS_ON("soundsOn"),
    THEME("theme"),
}
